packages/cli/test/commands/element.test.ts

```diff
@@ -1583,4 +1590,102 @@ Moving the specified elements to common.
       })
     })
   })
+
+  describe('fix command', () => {
+    const commandName = 'fix'
+    let cliArgs: mocks.MockCliArgs
+    let workspace: MockWorkspace
+    let fixElementsMock: jest.Mock
```

Comment:
we should used typed functions
```suggestion
    let fixElementsMock: jest.MockedFunction<typeof fixElements>
```